src/EnergyPlus/HeatBalanceManager.cc

```diff
@@ -1880,7 +1880,10 @@ namespace HeatBalanceManager {
                     ctf_dimensions = 1;
                 }
                 Real64 tube_spacing = fields.at("tube_spacing").get<Real64>();
-                Real64 calculation_position = fields.at("two_dimensional_temperature_calculation_position").get<Real64>();
+                Real64 calculation_position = 0.0;
+                if (fields.find("two_dimensional_temperature_calculation_position") != fields.end()) {
+                    calculation_position = fields.at("two_dimensional_temperature_calculation_position").get<Real64>();
+                }
```

Comment:
In the future, the new method to read the json inputs, so that this string is not searched for twice, is:

    Real64 calculation_position = 0.0;
    auto &twoDimTempCalcPos = fields.find("two_dimensional_temperature_calculation_position");
    if (twoDimTempCalcPos != fields.end()) {
        calculation_position = twoDimTempCalcPos.value().get<Real64>();
    }

Reply:
It seems find returns an iterator so drop the & and simply use auto without the reference. See [this comment](https://github.com/NREL/EnergyPlus/pull/9886/files/ab282fe748e9cc81440ba47348185d799379b825#r1136968086).

Reply:
Thanks, @rraustad. Just fixed it

Reply:
Looks good now.  Defaults to 0.0, then uses the input value if the user provided it.  Annoying that we have to duplicate the default value both in the IDD and the code, but this is fine for right now.